app/controllers/bookings_controller.rb

```diff
@@ -1,20 +1,23 @@
 class BookingsController < ApplicationController
-  before_action :set_booking, only: [:show, :edit, :update]
-  before_action :set_boat, only: [:new, :create]
+  before_action :find_booking, only: [:show, :edit, :update]
+  before_action :find_boat, only: [:new, :create]
   before_action :authenticate_user!, only: [:new, :create]
+  skip_before_action :authenticate_user!, only: [:index, :show]
 
   def index
-    @bookings = Booking.all
+    @bookings = policy(Booking).order(name: :asc)
```

Comment:
I think it is policy_scope(Booking). Bookings don't have a name, so I don't think we can order them alphabetically by name. I believe that will throw an error. Please let's test the code locally before we do the pull request.

Reply:
thank you. i corrected it by sorting by check-in date.